api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -31,11 +36,13 @@
 
   private static final Logger log = Logger.getLogger(BillingProjectBufferService.class.getName());
 
-  private static final int SYNCS_PER_INVOCATION = 5;
   private static final int PROJECT_BILLING_ID_SIZE = 8;
   @VisibleForTesting static final Duration CREATING_TIMEOUT = Duration.ofMinutes(60);
   @VisibleForTesting static final Duration ASSIGNING_TIMEOUT = Duration.ofMinutes(10);
-
+  private static ImmutableMap<BufferEntryStatus, Duration> statusToGracePeriod =
```

Comment:
Is the goal of this to clean things up that we think may be failed/errored in FireCloud?

Reply:
I keep hearing anecdotally things about leaked projects and the buffer getting in a bad state. I'm still not used to dealing with so many cron-based utilities, but I think the best thing to do in this situation is to over-log. We're definitely not dealing with such huge volumes of entries that space is a concern. Likewise, it's not going to matter in terms of performance since most of our time is spent waiting on Firecloud and/or GCP.